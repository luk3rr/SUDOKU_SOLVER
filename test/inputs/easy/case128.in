070024000 080307402 000060100 000870300 000041029 548290007 000000750 004000000 107000060 
