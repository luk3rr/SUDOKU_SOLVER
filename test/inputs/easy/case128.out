Solving the following grid:
0 7 0 | 0 2 4 | 0 0 0 
0 8 0 | 3 0 7 | 4 0 2 
0 0 0 | 0 6 0 | 1 0 0 
------+-------+------
0 0 0 | 8 7 0 | 3 0 0 
0 0 0 | 0 4 1 | 0 2 9 
5 4 8 | 2 9 0 | 0 0 7 
------+-------+------
0 0 0 | 0 0 0 | 7 5 0 
0 0 4 | 0 0 0 | 0 0 0 
1 0 7 | 0 0 0 | 0 6 0 


Solution found :')

3 7 9 | 1 2 4 | 5 8 6 
6 8 1 | 3 5 7 | 4 9 2 
4 2 5 | 9 6 8 | 1 7 3 
------+-------+------
9 1 2 | 8 7 6 | 3 4 5 
7 3 6 | 5 4 1 | 8 2 9 
5 4 8 | 2 9 3 | 6 1 7 
------+-------+------
8 9 3 | 6 1 2 | 7 5 4 
2 6 4 | 7 8 5 | 9 3 1 
1 5 7 | 4 3 9 | 2 6 8 

Algorithm: BFS
Total time: 11 ms
Total expanded states: 1939
